modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP Workload Identity cluster, you must use the `ccoctl` binary to generate the cloud credentials manually and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
```

Comment:
In our cert-manager document, you should share a link (or links) for OpenShift `GCP Workload Identity cluster` and for `ccoctl` extraction. Take 4.13 for example, all are in https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html , specifically [`ccoctl` extraction](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html#cco-ccoctl-configuring_cco-mode-gcp-workload-identity) is there.

Reply:
Hi, @xingxingxia I added the link for the "Configuring the Cloud Credential Operator utility" section in the additional resources section. Do we also need the link for the "Installing an OpenShift Container Platform cluster configured for manual mode with GCP Workload Identity" section in the additional resources section?

Reply:
Ah, I suppose yes!